packages/core/src/server/graphql/entity.ts

```diff
@@ -70,8 +70,8 @@ export const buildEntityTypes = ({
             fieldConfigMap[columnName] = {
               type: new GraphQLNonNull(
                 new GraphQLList(
-                  new GraphQLNonNull(entityGqlTypes[column.referenceTable])
-                )
+                  new GraphQLNonNull(entityGqlTypes[column.referenceTable]),
```

Comment:
Seems a bit strange to me that this line would have a comma. Is this what folks are doing with their lint rules these days?